transport/src/main/java/org/dousi/netty/NettyClient.java

```diff
@@ -94,6 +94,8 @@ private void processResponse(Response response) {
     } catch (InterruptedException i) {
       close();
       throw new DousiTransportException("NettyClient: connect().sync() interrupted", i);
+    } catch (Exception connectedFailed) {
+      throw new DousiException("Connected failed. ", connectedFailed);
```

Comment:
1. Catch Exception is too broad.
2. Use `DousiConnectionRefusedException` instead of `DousiException `;

Reply:
make sence